avocado/core/job.py

```diff
@@ -169,12 +171,14 @@ def _make_test_runner(self):
 
     def _set_output_plugins(self):
         for key in self.args.__dict__:
-            if key.endswith('_result'):
-                result_class = getattr(self.args, key)
+            result_class = getattr(self.args, key)
+            try:
                 if issubclass(result_class, result.TestResult):
                     result_plugin = result_class(self.view,
                                                  self.args)
                     self.result_proxy.add_output_plugin(result_plugin)
+            except TypeError:
+                pass
```

Comment:
RFC: if some class is indeed a plugin (an output plugin here) and it fails to instantiate, should we log that here also? Does this get propagated into the `ErrorsLoading` list and then into the UI?